src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.js

```diff
@@ -47,6 +48,8 @@ const AnchorRenderer = (props) => {
         // If we are handling a New Expensify link then we will assume this should be opened by the app internally. This ensures that the links are opened internally via react-navigation
         // instead of in a new tab or with a page refresh (which is the default behavior of an anchor tag)
         if (internalNewExpensifyPath) {
+            const reportID = attrPath.split('r/').pop();
```

Comment:
What if you have `new.expensify.com/r/123/participants` etc links? this wont work

Reply:
true true, fixing